source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -127,16 +127,24 @@ AndroidContext: class extends OpenGLContext {
 			(rasterResult, fenceResult) = super(gpuImage)
 		(rasterResult, fenceResult)
 	}
+	_unpack: static func (source: GpuImage, target: GpuImage, map: GpuMap, targetWidth: Int, transform: FloatTransform3D, scaleX: Float, scaleY: Float, startY: Float) {
+		map add("texture0", source)
+		map add("targetWidth", targetWidth)
+		map add("transform", transform)
+		map add("scaleX", scaleX)
+		map add("scaleY", scaleY)
+		map add("startY", startY)
+		target canvas draw(source, map)
+	}
 	unpackBgraToYuv420Semiplanar: func (source: GpuImage, targetSize: IntSize2D) -> GpuYuv420Semiplanar {
 		target := this createYuv420Semiplanar(targetSize) as GpuYuv420Semiplanar
-		this _unpackRgbaToMonochrome targetSize = target y size
-		this _unpackRgbaToMonochrome sourceSize = source size
-		this _unpackRgbaToMonochrome transform = FloatTransform3D createScaling(source transform a, -source transform e, 1.0f)
-		target y canvas draw(source, this _unpackRgbaToMonochrome)
-		this _unpackRgbaToUv targetSize = target uv size
-		this _unpackRgbaToUv sourceSize = source size
-		this _unpackRgbaToUv transform = FloatTransform3D createScaling(source transform a, -source transform e, 1.0f)
-		target uv canvas draw(source, this _unpackRgbaToUv)
+		sourceSize := source size
+		transform := FloatTransform3D createScaling(source transform a, -source transform e, 1.0f)
+		This _unpack(source, target y, this _unpackRgbaToMonochrome, targetSize width, transform, (targetSize width as Float) / (4 * sourceSize width),
+			targetSize height as Float / sourceSize height, 0.0f)
```

Comment:
(targetSize width as Float) / (4 * sourceSize width),
targetSize height as Float / sourceSize height, 0.0f)
```
uses parentheses inconsistently. Also, this would look better with those two parameters on the same line.